tests/aws/aws_kubernetes.rs

```diff
@@ -20,39 +20,70 @@ use std::io::{BufReader, Read};
 use std::path::Path;
 use std::process::Command;
 use std::{env, fs};
-use test_utilities::aws::AWS_KUBERNETES_VERSION;
+use test_utilities::aws::{
+    aws_access_key_id, aws_default_region, aws_secret_access_key, AWS_KUBERNETES_VERSION,
+};
 
-pub const QOVERY_ENGINE_REPOSITORY_URL: &str = "CHANGE-ME";
+pub const QOVERY_ENGINE_REPOSITORY_URL: &str = "https://github.com/Qovery/engine.git";
 pub const TMP_DESTINATION_GIT: &str = "/tmp/qovery-engine-master/";
 pub const GIT_LOGIN: &str = "CHANGE-ME";
 pub const GIT_TOKEN: &str = "CHANGE-ME";
 
 #[test]
-#[ignore]
-fn create_and_upgrade_cluster_from_master_branch() {
+fn upgrade_new_cluster() {
     init();
+    // create a cluster with last version of the engine
     let tmp_dir = format!("{}{}", TMP_DESTINATION_GIT, generate_id());
     let current_path = env::current_dir().unwrap();
     fs::remove_dir_all(TMP_DESTINATION_GIT);
     let gr = GitRepository {
         url: QOVERY_ENGINE_REPOSITORY_URL.to_string(),
-        credentials: Some(Credentials {
-            login: GIT_LOGIN.to_string(),
-            password: GIT_TOKEN.to_string(),
-        }),
+        // this repo is public !
+        credentials: None,
         commit_id: "".to_string(),
         dockerfile_path: "".to_string(),
     };
     fs::create_dir_all(tmp_dir.clone());
-    info!("Cloning qovery-engine repository");
+    info!("Cloning old engine repository");
```

Comment:
Why don't you just copy/paste the current one and checkout on the main branch instead?

Reply:
will put a enhancement on it